sdk/golang-example/golang-projects/fibonacci.txt

```diff
@@ -0,0 +1,23 @@
+   // Simple Program to calculate fibonacci of input number
+   
+	package main
+	import "fmt"
+	func FibonacciRecursion(n int) int 
+	{
+ 		if n <= 1 
+ 		{
+		return n
+		}
+		return FibonacciRecursion(n-1) + FibonacciRecursion(n-2)
+	}
+	
+	func main()
+	{
+		fmt.Print("Enter number : ")
+		var n int
+		fmt.Scanln(&n)
```

Comment:
I am not sure this line can be executed in Wasmtime, which is mentioned in the readme you provide, if you compile this Go program to WASM with the WASI backend.

Separately, can you change the file extend name to the Golang file name, for example, `fibonacci.go` and apply the go fmt convention. Particularly, remove the extra heading white space.